config/deploy.js

```diff
@@ -33,6 +33,8 @@ module.exports = function(deployTarget) {
     ENV["s3-index"].region = 'ap-southeast-1';
     ENV.cloudfront = {
       distribution: 'E2TZ3PLCL45IAQ',
+      accessKeyId: process.env.AWS_SECRET_ACCESS_KEY,
```

Comment:
is that a copy-paste error or is it supposed to use the same env var for both properties?

Reply:
thanks, it's a copy 🍝 error 🙈